### PR TITLE
coffee house project

### PR DESCRIPTION
**Pull request by ashraffares**
Add home page contains a heading for the coffee house name and caption.
Add menu page contains nine cards each card has  (an image -  a name - a price ).
Add contact page contains (contact form - about - map).
and all pages have a navbar with three links to navigate between pages.

used Webpack assets (CSS - Images)
Dom manipulation

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hello @ashraffares  👋

**You've done pretty well here, well structured codebase, and a good readme. 👌**

However, let's make a couple of changes.

- [x] Kindly deploy the project online and include the live link in the readme.

- [x] Please ensure you use the same eslint config rules as stated in these [link](https://github.com/microverseinc/linters-config/blob/master/javascript/.eslintrc.json). It helps to maintain professionalism in the code.

- Please refer to other comments on the code.

### **STATUS: CHANGES REQUESTED** 👷‍♂️

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

**HAPPY CODING 🎉💻🎉**

**Michgolden Ukeje (CR)**

File: src/index.js [hunk 1]
```diff
@@ -1,3 +1,46 @@
...
+  const nav = document.createElement('nav');
+  nav.classList.add('nav');
+  const ul = document.createElement('ul');
+  ul.classList.add('ul');
+
+  const links = ['Home', 'Menu', 'Contact'];
+  for (let i = 0; i < 3; i += 1) {
+    const btn = document.createElement('buttton');
+    btn.innerHTML = links[i];
+    btn.classList.add('li', links[i]);
```
Comment (mikenath223) [inline src/index.js]: - [x] Please change these and all other regular functions to es6 arrow function syntax.

File: src/css/style.css [hunk 2]
```diff
@@ -7,7 +7,7 @@ body {
   background-image: url("../img/mainbg.jpg");
   background-repeat: no-repeat;
   background-size: 100% 100%;
-  height: auto;
+  height: 664px;
```
Comment (mikenath223) [inline src/css/style.css]: - [ ] On the last line in the stylesheet there is an `!important` tag. Kindly remove it avoiding CSS bad practices.

File: src/js/contact.js [hunk 3]
```diff
@@ -1,19 +1,4 @@
...
-    const li = document.createElement('li');
-    li.innerHTML = links[i];
-    li.classList.add('li');
-    ul.appendChild(li);
-  }
-
-  nav.appendChild(ul);
-
   const mapouter = document.createElement('div');
   mapouter.classList.add('mapouter');
```
Comment (mikenath223) [inline src/js/contact.js:3]: - [x] The following properties below are currently deprecated in iframes. Kindly remove them:
```
  iframe.frameBorder = '0';
  iframe.scrolling = 'no';
  iframe.marginHeight = '0';
  iframe.marginWidth = '0';
```

**Review by arslanbisharat — APPROVED** [review]
#Approved 👏 

Hi @ashraffares, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**